src/components/LooMap/ToiletMarkerIcon.tsx

```diff
@@ -1,13 +1,20 @@
 import L from 'leaflet';
 
-const ICON_DIMENSIONS = [22, 34];
-const LARGE_ICON_DIMENSSIONS = ICON_DIMENSIONS.map((i) => i * 1.5);
-const getIconAnchor = (dimensions: number[]) => [
-  dimensions[0] / 2,
-  dimensions[1],
+const ICON_DIMENSIONS: [number, number] = [22, 34] as const;
```

Comment:
This is widened to the number double. For `as const` to work it must be paired with `satisfies`.

https://chrisvaillancourt.io/posts/combining-typescript-satisfies-and-const-assertion/